styles/desktop.css

```diff
@@ -0,0 +1,352 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap');
+
```

Comment:
- [ ] as requested by the previous reviewer kindly consider making sure that as you build your desktop version  your mobile version still looks like expected cause currently your cards are disturbed check below 
- your design 👇 
<img width="430" alt="Screen Shot 2023-04-07 at 13 01 40" src="https://user-images.githubusercontent.com/60146030/230598135-4b96491a-02f3-41f2-9679-7c7892fb3bd7.png">

- figma design 👇 
<img width="442" alt="Screen Shot 2023-04-07 at 13 04 30" src="https://user-images.githubusercontent.com/60146030/230598418-596aaf51-5705-4b54-9eab-f9ee43a35e9c.png">

- your design 👇 
<img width="334" alt="Screen Shot 2023-04-07 at 13 05 32" src="https://user-images.githubusercontent.com/60146030/230598816-cf0356ef-1af8-472e-a535-3995d78272be.png">

- figma design 👇 
<img width="424" alt="Screen Shot 2023-04-07 at 13 09 19" src="https://user-images.githubusercontent.com/60146030/230599232-686f0589-04cf-4bf5-9117-4714ed463b89.png">


- kindly fix that and your project will be good to go and that will make your project an exact duplicate of the provided design

Reply:
Hi @divinecharlotte 

Thank you for your review but as you can see in my comment I clearly made it plain that this branch is for my desktop version. It looks to me like you guys(previous reviewers) are not sticky to the current task ahead of you, and therefore concerning yourself with my mobile version. If I quote the objective correctly you are to review my desktop version, not my mobile version. I will fix my mobile version but that's not the task you should be concerned with as a reviewer of this specific pull request.

Thank you😎

Reply:
Hi @Paulinelali, it's also part of our job to check that the mobile version still looks as expected. 😃